analysis_options.yaml

```
@@ -25,4 +25,4 @@ linter:
     # prefer_single_quotes: true  # Uncomment to enable the `prefer_single_quotes` rule
 
 # Additional information about this file can be found at
-# https://dart.dev/guides/language/analysis-options
+# https://dart.dev/guides/language/analysis-options
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)